### PR TITLE
DocumentationViewer: Add missing docs from gtk-doc

### LINKED ISSUE #373: Investigate missing docs
Some docs are missing from the documentation viewer.

The ones I could think of: `GObject`, `Gio`, `Webkit`

As far as I can tell we should show all the docs that

`flatpak run --runtime-version=44 --nofilesystem=host org.gnome.Devhelp` shows.

### PR DESCRIPTION
**Pull request by AkshayWarrier**
Closes #373

### CODE REVIEW

File: src/DocumentationViewer.js [hunk 1]
```diff
@@ -87,6 +87,10 @@ export default function DocumentationViewer({ application }) {
   listbox.set_sort_func(sort);
   listbox.invalidate_sort();
   listbox.set_filter_func(filter);
+  listbox.connect("row-selected", (self, row) => {
+    webview.load_uri(row.uri);
+  });
```
Comment (sonnyp) [inline src/DocumentationViewer.js:92]: this didn't need to be executed for each doc

File: src/DocumentationViewer.js [hunk 2]
```diff
@@ -97,14 +101,11 @@ export default function DocumentationViewer({ application }) {
...
-      listbox.connect("row-selected", (self, row) => {
-        webview.load_uri(row.uri);
-      });
     })
     .catch(logError);
 
-  function sort(row1, row2) {
-    return row1.title > row2.title;
+  function sort(a, b) {
+    return a.title.localeCompare(b.title);
```
Comment (sonnyp) [inline src/DocumentationViewer.js:108]: sort case insensitive

File: src/DocumentationViewer.js [hunk 3]
```diff
@@ -125,25 +126,22 @@ export default function DocumentationViewer({ application }) {
...
   for (const dir of dirs) {
-    try {
-      const results = await Promise.allSettled([
-        readDocIndexJSON(base_path, dir),
-        readDocIndexHTML(base_path, dir),
-      ]);
-
-      let namespace = results.filter((result) => result.status === "fulfilled");
-      if (namespace.length) namespace = namespace[0].value;
-      else continue;
```
Comment (sonnyp) [inline src/DocumentationViewer.js]: confusing :)

**Review by sonnyp — APPROVED** [review]
neat, I made some improvements please have a look and let me know if something isn'tt clear https://github.com/sonnyp/Workbench/pull/394/commits/aaac9c510b2fb1231ce081ca75050cdab5160ed1

File: src/DocumentationViewer.js (hunk 2 above)
Comment (AkshayWarrier) [reply src/DocumentationViewer.js:108]: Good call

**Pull request merged**